exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,18 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
```

Comment:
Just to let you know if you if block contains just one line, you don't have to use {}, You can write this code shorter this way like :
 ```
 if ((mark) >= 80) return "A";
  else if (mark < 80 && mark > 60) return "B";
  else if (mark < 60 && mark > 50) return "C";
  else return "F";
}

```